websites/S/SOAP2DAY/dist/metadata.json

```diff
@@ -8,9 +8,20 @@
   "description": {
     "en": "You can watch all videos without logging in or registration. Soap2day is free all the time, but there are few ads on videoplayer page."
   },
-  "url": "soap2day.to",
+  "url": [
+    "soap2day.to",
+    "soap2day.ac",
+    "soap2day.sh",
+    "soap2day.cx",
+    "s2dfree.to",
+    "s2dfree.cc",
+    "s2dfree.de",
+    "s2dfree.is",
+    "s2dfree.nl",
+    "secretlink.xyz"
+  ],
```

Comment:
These aren't used as fallback on current extension 
https://github.com/PreMiD/Extension/blob/release/src/util/tabPriority.ts#L56-L82

Reply:
Will remove the regex then as this is safer